Leadership.md

```diff
@@ -31,6 +31,7 @@ Leadership at Octopus is all about positively influencing others. We do this usi
 - I have a proven track record of deriving the best outcomes from difficult conversations over all mediums.
 - I effectively tailor the content of my conversations to my audience.
 - People seek my input to validate and improve their own communications.
+- I facilitate healthy conflict through negotiation to arrive at positive, constructive outcomes.
```

Comment:
If we agree on this one, I plan to share wider in management channel to give others a chance for feedback here too as the Leadership capabilities spread further than us.

Reply:
Nice 👍 As a data point: I've found the concept of seeking to create "healthy conflict" really helpful in discussions with a few leaders at Octopus, so I'm supportive of getting it into Octopus People 👍

Reply:
+1 to @Waldo000000 comments

Reply:
@corey.underdown there is similar wording in the Senior Engineer role description - manage difficult conversations and tactfully challenge others.
&
I spotted a contentious issue that could have gone badly and facilitated everyone toward a decision that resolved the situation.
Is that useful or reusable?

Reply:
Thanks Liam, both of those are similar in what we're going for, with maybe just some slight differences that `facilitate healthy conflict` might articulate better.  

> manage difficult conversations and tactfully challenge others

I think here the "difficult conversations" is slightly different to what we're going for as I don't always see it as a difficult conversation -> healthy conflict.  

> I spotted a contentious issue that could have gone badly and facilitated everyone toward a decision that resolved the situation.

This one I feel misses letting the healthy conflict help shape better work, and instead might circumvent it early to avoid a bad conflict, instead of leading it towards a heathy conflict. 

I appreciate that wording is incredible difficult and there's really tiny differences between things 😂 Please let me know if you disagree, but I've seen a lot of 👍 from people on this wording so I'm inclined to go with it for now, and reflect again in the future if we need to adjust the wording :)